malaria-ios/ViewControllers/Game.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+class Game: NSObject {
```

Comment:
Is this working? It's because, if this is stored in the CoreData then I believe it is missing the @NSManaged tag before the instance variables like so:

```
@NSManaged var numberOfLevels: Int64
@NSManaged var name: String
```

You can generate the class from the model itself.